argo-cd-apps/base/tenants-config/tenants-config.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: tenants-config
+spec:
+  generators:
+    - git:
+        repoUrl: https://github.com/redhat-appstudio/tenants-config
+        revision: main
```

Comment:
should be pinned to specific version

Reply:
We  don't want to pin it to specific version. We want ArgoCD to track main for this application.

Reply:
None of pull-requests in that project has been reviewed before merging, so I do not trust that project.

Reply:
(but not blocking this PR)

My understading is that we do not want to have some random unreviewed changed, so that's why we are always pinning to specific sha.

CC: @sbose78 @arewm

Reply:
pr will get reviewd on https://github.com/redhat-appstudio/tenants-config
I just added a simple directory structure to it so we can get things rolling.

Reply:
new PR with required review: https://github.com/redhat-appstudio/tenants-config/pull/4